packages/fast-usdc-contract/src/fast-usdc.contract.ts

```diff
@@ -187,8 +194,7 @@ export const contract = async (
     getNobleICA,
     vowTools: tools.vowTools,
     zcf,
-    // UNTIL we have an generic way to attenuate an Exo https://github.com/Agoric/agoric-sdk/issues/11309
-    chainHub: { resolveAccountId: chainHub.resolveAccountId.bind(chainHub) },
```

Comment:
please note there is actually an upgrade hazard here which is hidden in endowments and not well documented: `chainHub` previous to this change *should* have been wrapped as a record containing a `resolveAccountId` function, which is itself getting an automatic exo definition. For some reason [the baggage](https://github.com/Agoric/agoric-sdk/commit/bcecfdb1999672b55f7f366e6a1c8c2ba767e039#diff-f421976e2420c422682469693b764716d7b92834eb59649e3360d6d21c6328ebR3205-R3214) is not showing this, and I don't understand why.

By changing the definition here, we are effectively removing the need for this wrapping, which would fail an upgrade of the contract as the kind would no longer be redefined. Since this hasn't been deployed yet, this is fine.

Reply:
Wow! I had no idea.

> which is itself getting an automatic exo definition

Where is this code?

> Since this hasn't been deployed yet, this is fine

Whew!

Reply:
> effectively removing the need for this wrapping

Is there some wrapping code we could remove?

Reply:
> Where is this code?

That's in [endowments.js](https://github.com/Agoric/agoric-sdk/blob/master/packages/async-flow/src/endowments.js) that you wrote ;)

Reply:
Thanks! Looking

Reply:
Ok I got confused, `chainHub` is not used as an asyncFlow endowment, but as a closed over capability for an exo. As such there is no upgrade hazard, and that explains why it wasn't in the baggage snapshot.

Please proceed.

Reply:
Yeah, ChainHub is ephemeral.

> IMO, this PR is net positive, but I don't know of any objective and affordable way to test that. Reviewers, what do you think?

I don't think it's positive. It encumbers ChainHub in the baggage's kinds. 

My XXX was to use something like `attenuate` that could permit inherited properties.